.evergreen/scripts/generate-tasks.js

```diff
@@ -153,36 +81,34 @@ const generateTasks = () => {
       : targetsFromChangedFiles(changes);
 
   const tasks = [];
-  const buildvariants = targets.map((bv) => {
+  const buildvariants = [];
+
+  targets.forEach((bv) => {
     const bvTasks = [];
     const displayTasks = [];
     TASK_MAPPING[bv].forEach((name) => {
       if (name === Tasks.E2EParallel) {
```

Comment:
> Note: generator tasks cannot be run on the same patch.

Should we consider removing `e2e_parallel` from `generate-tasks` and perhaps only having it managed by the new generator? Then opening a PR could invoke both.

Reply:
done, here's the patch: https://spruce.mongodb.com/version/66f2d29112158a00072c45f5/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC

It does not run on this PR as I haven't edited the project settings